### DISCUSSION TITLE
If I mount an external library without :ro can I then delete pictures form there?

### DISCUSSION THREAD

**Discussion by dffvb** [immich-app/immich#4215] [Q&A] [answered]
Title basiccally says it, I want to be able to delete pictures from existing external libraries. Would simply leavon :ro away enable the delte button?

**Comment by traktuner** [comment, marked as answer]
Hello,
no, this is not possible. Immich does not touch external files in any way. The :ro just "makes sure" that the code can not touch those files. It also depend on permissions - Immich can only edit/delete files it owns.
In my case, the permission set of the immich docker app and the contents of my external library are completely different. So Immich can only read those files from a system level perspective, even if the docker container wants to delete the files - it can't.
Hope that helps!

**Reply by dffvb** [reply]
Thank you - bummer.

**Comment by cocciasecca** [comment]
Hello there, sorry for reopening the discussion, but what if I setup the library with :ro and later, at some point, I decide to give Immich permission to read and write. Will it be able to delete images then?

**Reply by bo0tzz** [reply]
Yes
